src/game/behaviors/capswitch.inc.c

```diff
@@ -6,12 +6,9 @@ void cap_switch_act_0(void) {
     cur_obj_scale(0.5f);
     o->oPosY += 71.0f;
     spawn_object_relative_with_scale(0, 0, -71, 0, 0.5f, o, MODEL_CAP_SWITCH_BASE, bhvCapSwitchBase);
-    if (gCurrLevelNum != LEVEL_UNKNOWN_32) {
-        if (SM64AP_PressedSwitch(D_8032F0C0[o->oBehParams2ndByte])) {
-            o->oAction = 3;
-            o->header.gfx.scale[1] = 0.1f;
-        } else
-            o->oAction = 1;
+    if (gCurrLevelNum != LEVEL_UNKNOWN_32 && SM64AP_PressedSwitch(D_8032F0C0[o->oBehParams2ndByte])) {
```

Comment:
Where does `gCurrLevelNum != LEVEL_UNKNOWN_32` come from?

Reply:
This check is based on the original from [upstream](https://github.com/sm64pc/sm64ex/blob/nightly/src/game/behaviors/capswitch.inc.c#L8-L15)

Reply:
Thats ok, but then keep their syntax as well

Reply:
By that do you mean I should revert this change?
https://github.com/N00byKing/sm64ex/pull/6/commits/e1e43965521e1d2d43378ee11cd920bbdcfe250d

Reply:
ye

Reply:
Fixed in https://github.com/N00byKing/sm64ex/pull/6/commits/334f11e08c7e2835da9f99388701a405aa191514